src/client/datascience/kernel-launcher/kernelLauncher.ts

```diff
@@ -51,21 +50,6 @@ export class KernelLauncher implements IKernelLauncher {
         @inject(IKernelDependencyService) private readonly kernelDependencyService: IKernelDependencyService
     ) {}
 
-    // This function is public so it can be called when a test shuts down
```

Comment:
Why did you remove this? Is it not called anywhere?

Reply:
Yes, not called anywhere